File: DSA/9. Palindrome Number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        long rem=0;
        long sum=0;
        int temp=x;
        if(x<0){
            return 0;
        }
        else{
                while(temp!=0){
                    rem=temp%10;
                    temp=temp/10;
                    sum=sum*10+rem;
                
                }
                if(sum==x){
                    return 1;
                }

        }
        return 0;
        
    }
};

File: DSA/9. Palindrome Number.java
class Solution {
    public boolean isPalindrome(int x) {
        if(x<0) return false;
        int temp=0,sum=0,num=x;
        while (num!=0){
            sum= sum*10+num%10;
            num=num/10;
        }
        if(sum==x) return true;
        return false;
    }
}